### PR TITLE
setsid: add page

### PR DESCRIPTION
**Pull request by git-em**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
`setsid from util-linux 2.37.4`

### CODE REVIEW

**Comment by git-em** [comment]
Listed on #2214

File: pages/linux/setsid.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+> The created session is by default not controlled by the current terminal.
+> More information: <https://manned.org/setsid>.
+
+- Run a program in a new session:
+
+`setsid {{program}}`
+
+- Run a program in a new session discarding the resulting output and error:
+
+`setsid {{program}} > /dev/null 2>&1`
```
Comment (EmilySeville7cfg) [inline pages/linux/setsid.md:13]:
```suggestion
`setsid {{program}} > &> /dev/null`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/linux/setsid.md (hunk 1 above)
Comment (git-em) [reply pages/linux/setsid.md:13]: I initially written like this, but when i tested it in `dash` it didn't work, so i opted for the most shell agnostic and maybe POSIX compliant way (if i understood [this](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07_02) well).

Comment (git-em) [reply pages/linux/setsid.md:13]: Just as a curiosity, neither seemed to work with `[t]csh`...

File: pages/linux/setsid.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Run a program creating a new process:
+
+`setsid --fork {{program}}`
+
+- Return the exit value of a program as the return value of setsid when the program exits:
```
Comment (pixelcmtd) [inline pages/linux/setsid.md:19]:
```suggestion
- Return the exit code of a program as the exit code of setsid when the program exits:
```

File: pages/linux/setsid.md (hunk 1 above)
Comment (pixelcmtd) [inline pages/linux/setsid.md:13]:
```suggestion
`setsid {{program}} >/dev/null 2>&1`
```
I prefer this, I've never seen `&>` and don't know what it does neither how portable it is.

Comment (git-em) [reply pages/linux/setsid.md:13]: I believe the space between the "`>`" and the following text is good practice, as seen on this [page](https://www.gnu.org/software/bash/manual/html_node/Redirections.html) about redirection in `bash`. 

As for the `&>`, it's a convenient way to redirect stdout and stderr, present in `bash` ([see previous link](https://www.gnu.org/software/bash/manual/html_node/Redirections.html)) and [`zsh`](https://zsh.sourceforge.io/Doc/Release/Redirection.html), but not in some other shells.

**Review by CleanMachine1 — APPROVED** [review]
Once @EmilySeville7cfg's review is done, please merge

File: pages/linux/setsid.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/linux/setsid.md:13]: Yes, maybe it's better to leave your code unchanged because `Tl;Dr` is not restricted to concrete shells.

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Pull request merged**